src/main/Config.cpp

```diff
@@ -1679,5 +1744,12 @@ Config::toString(SCPQuorumSet const& qset)
     return fw.write(json);
 }
 
+const std::vector<std::chrono::microseconds>
```

Comment:
I think you ment to return a const reference

Reply:
Yes. Fixed it!